redash/app.py

```diff
@@ -0,0 +1,62 @@
+from flask import Flask
+from flask_sslify import SSLify
+from werkzeug.contrib.fixers import ProxyFix
+
+from . import mail, settings
+from .utils import routes
+
+
+class Redash(Flask):
```

Comment:
I basically tried to separate Flask app specific code in the flask.Flask subclass and all Redash specific code in the `create_app` function.